lib/template.js

```diff
@@ -1095,9 +1096,17 @@ class Template {
                 message = `${message}: ${allowedValues.join(', ')}`;
```

Comment:
Should allowedValues move to details property as well?  Any variable length string in the message property could lead to formatting issues in the CM UI.

Reply:
Good point! I have pushed a change to allowedValues as well.